pyproject.toml

```diff
@@ -10,6 +10,10 @@ extend-exclude = '''
 )
 '''
 
+[build-system]
+requires = ["setuptools>=42", "wheel", "pybind11~=2.10.1"]
```

Comment:
```suggestion
requires = ["setuptools>=62", "wheel", "pybind11~=2.10.1"]
```

Reply:
We did want to change the Pybind version though, to a minimum that we measured right?

Reply:
It looks like [2.7.0](https://pybind11.readthedocs.io/en/stable/changelog.html#v2-7-0-jul-16-2021) has `intree_extension` which is what we use. I will give it a try.

Reply:
setuptools: https://github.com/PennyLaneAI/catalyst/pull/99/commits/a4443b34b9c5c1da0dee54395fe843b199de19f8, pybind: https://github.com/PennyLaneAI/catalyst/pull/99/commits/1479547c35eebfe2dd57fda75ab9119885075120